utils/api_checker/CMakeLists.txt

```diff
@@ -1,5 +1,6 @@
 set(framework "FrameworkABIBaseline")
 swift_install_in_component(FILES "swift-api-checker.py"
+                           PERMISSIONS OWNER_READ OWNER_WRITE OWNER_EXECUTE GROUP_READ GROUP_EXECUTE WORLD_READ WORLD_EXECUTE
```

Comment:
You want `swift_install_in_component(PROGRAMS` instead of `FILES`. Then you don't need this bit.